Classes/Private/Manager/Audio/OCTAudioEngine.m

```diff
@@ -7,10 +7,12 @@
 //
 
 #import "OCTAudioEngine.h"
+#import "TPCircularBuffer.h"
 @import AVFoundation;
 
 static const AudioUnitElement kInputBus = 1;
 static const AudioUnitElement kOutputBus = 0;
+static const int kBufferLength = 1025;
```

Comment:
No particular reason, I will probably have to change it later. Simply a placeholder for now. Do you have a number in mind?